.github/workflows/gh-ci.yaml

```diff
@@ -103,10 +103,9 @@ jobs:
         conda info
         conda list
 
-
     - name: Run tests
       run: |
-        pytest -n auto -v --cov=basicrta --cov-report=xml --color=yes basicrta/tests/
+        pytest -n auto -v --cov=basicrta --cov-report=xml --color=yes basicrta/tests/ --basetemp=testtemp
```

Comment:
Is ` --basetemp=testtemp` needed? 

If not necessary, I'd leave it out (especially so that in CI the best temp dir is chosen)

Reply:
I used it while testing and forgot to remove it, I've removed it now.